package.json

```diff
@@ -5,6 +5,9 @@
   "author": "Douglas Christopher Wilson <doug@somethingdoug.com>",
   "license": "MIT",
   "repository": "pillarjs/finalhandler",
+  "type": "commonjs",
+  "main": "index.js",
+  "types": "index.d.ts",
```

Comment:
```suggestion
```

Reply:
At least in my PR `attw` resolves them without `types` in `package.json`